src/gordon_gcp/plugins/janitor/gpubsub_publisher.py

```diff
@@ -238,16 +238,21 @@ def publish(self, message):
         Once ``None`` is received from the channel, finish processing
         records and clean up any outstanding tasks.
         """
+        context = {'plugin': 'gpubsub-publisher'}
+        timer = self.metrics.timer('janitor-plugin-runtime', context=context)
+        await timer.start()
         while True:
             change_message = await self.changes_channel.get()
             if change_message is None:
                 break
-            # TODO (lynn): emit metric of message received once
-            #              aioshumway is released
+            await self.metrics.incr('janitor-msg-recv', context=context)
```

Comment:
`msg-recv` could be confused with receiving a pubsub message. Maybe `change-msg-recv`?